packages/backend/src/models/SpaceModel.ts

```diff
@@ -846,6 +851,7 @@ export class SpaceModel {
                             hasDirectAccess: !!user_with_direct_access,
                             inheritedRole: highestRole.role,
```

Comment:
I believe this means that we don't use `inheritedRole` anymore. 
Could you update the `inheritedRole` to be `highestProjectRole?.role` or something similar? We don't need `projectRole` being added I think 🤔

Reply:
I am seeing `inheritedRole` being used in `packages/frontend/src/components/common/ShareSpaceModal/ShareSpaceUserList.tsx` 😞

Reply:
oops! ignore!